a8cteam51/plugin-autoupdate-filter
# Remove WordPress hooks from the class constructor

**Changes**

Related to #12 

* Removed the class constructor from each of the plugin classes
* Replaced constructor with an `init()` method that contains all WordPress hooks
* Run method after instantiating the class

This separates out WordPress functionality from the class constructor. In this case, I don't actually believe we need a constructor.

**Testing**

These changes should not alter the function of the plugin in any way. To test:

* Install Plugin Autoupdate Filter on a site with out of date plugins
* When plugin auto-update schedule runs (twice daily run by default), confirm that plugins were updated successfully
* Confirm emails are sent to concierge@ email address
